### DISCUSSION TITLE
When will the v-calendar component of vuetify3 coming?

### DISCUSSION THREAD

**Discussion by skyline523** [vuetifyjs/vuetify#17213] [Q&A]
When will the v-calendar component of vuetify3 coming?

**Comment by simenkf** [comment]
Have you checked out Labs? Eta Q2-23 for Labs.

https://vuetifyjs.com/en/labs/introduction/

**Comment by danbaechtold** [comment]
I hate to nag, but started a client project months ago, expecting (just a basic) v-calendar in labs in Q1.
Then held my breath the whole Q2.
Now it's Q3.. 

What should I do? I have an appointment with the client in literally 1h..

**Reply by torerik123** [reply]
Same situation here. Would be really nice to get this released soon. I have a big project that will rely heavily on v-calendar.

**Comment by happymago** [comment]
Same here. Is there any update on when calendar will come out?

**Comment by sn0rcha** [comment]
Same here, this is the only thing stopping me from using Vuetify

**Comment by brianlangeland** [comment]
Going to jump on this as well. I really want to use Vuetify for an application, and one of the central pieces of the application will be the calendar within it... but alas, this component keeps getting pushed back.

**Comment by 1Luc1** [comment]
the corresponding Task: https://github.com/vuetifyjs/vuetify/issues/13469
and Draft: https://github.com/vuetifyjs/vuetify/pull/16803
latest reply from devs: https://github.com/vuetifyjs/vuetify/pull/16803#issuecomment-1759746878
